### DISCUSSION TITLE
Hosting Elsa 3.0 ElsaStudioBlazerWasm on IIS - 404 for login page

### DISCUSSION THREAD

**Discussion by priyankamohile** [elsa-workflows/elsa-core#4628] [Q&A]
Hi,

We've integrated Elsa 3.0 Server + Elsa 3.0 ElsaStudioBlazerWasm to create a few workflows. So far, everything runs okay with the dashboard and server apps running on localhost on their respective ports.
We now wish to host both of these apps in IIS. We've achieved success with the Server app, however, the dashboard app throws the below exception -

Failed to load resource: the server responded with a status of 404 () for the below URL

https://mywebsite/elsadashboard/login

We've changed the base URL on the index.html to the following, which helped to load all of the js scripts on the client side -

base href="/elsadashboard" 

However, our app is still breaking while trying to redirect to the login page. Is there any other change we'd need to make? Does anyone have a set of steps they followed, to host this successfully?

Any help would be much appreciated! :)

**Comment by code-abhijit** [comment]
@priyankamohile, 
I think you have hosted the Elsa Studio Blazor WASM application under "Default Web Site" in IIS. Changing the `<base href="/elsadashboard"/>` will resolve most of the frontend resources but some of the frontend stuff thinks the application is running out of the root folder and continues to use relative path.

One solution to this would be to host the application as a separate website, not under the "Default Web Site" in IIS. You will have to change the base url back to `<base href="~/"/>`

Hope this helps.
